### PR TITLE
add reading-time@1.0.0 w/ git auto-update

### LINKED ISSUE #9950: [Request] Add readingTime
**Library name:** readingTime
**Git repository url:** https://github.com/michael-lynch/reading-time.git
**npm package url(optional):**  
**License(s):** MIT
**Official homepage:** https://github.com/michael-lynch/reading-time
**Wanna say something? Leave message here:** n/a

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by lcd78706**
close #9950, cc @michael-lynch

Pull request for issue: #9950 
Related issue(s): # #

@x09326  Please review this PR. Thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/michael-lynch/reading-time
 * Official website (optional, not the repository): 
 * NPM package url (optional):
 * License and its reference: MIT  https://github.com/michael-lynch/reading-time/commit/14e894e9a1b0457fcbe9f0593f52f2ea432db9f5#diff-cab473f52840eb83e6884fc8712938dfL9
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 402
   - Count of watchers: 10
   - Count of forks: 61
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by lcd78706** [comment]
@x09326  I have fixed it.

File: ajax/libs/reading-time/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "readingTime.js"
```
Comment (x09326) [inline ajax/libs/reading-time/package.json]: @lcd78706 
I think use `"readingTime*"` is more flexible.

Comment (lcd78706) [inline ajax/libs/reading-time/package.json]: @x09326 OK. I have modified it.

**Comment by x09326** [comment]
LGTM
@cdnjs/intern2 please help me review it, thanks.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Nice work.

**Pull request merged**